io_scene_niftools/modules/nif_import/animation/transform.py

```diff
@@ -186,6 +213,9 @@ def import_keyframe_controller(self, n_kfc, b_obj, bone_name=None, n_bone_bind_s
                 scales = zip(times, scale_temp)
             # Bsplines are Bezier curves
             interp_rot = interp_loc = interp_scale = "BEZIER"
+        elif isinstance(n_kfc, NifFormat.NiMultiTargetTransformController):
+            # not sure what this is used for
```

Comment:
`For a single object use NiTransformController or you can array several objects you want to tranform in a NiMultiTargetTransformController. `

Reply:
Whatever that means lol. It's not used at the moment and it imports just fine, so I don't really get it xD

Reply:
I think its the ability to reuse the same animation, or have it affect multiple objects

Reply:
I see, so basically re-using keys to save space for several targets. We'll need a good example nif where this is actually used properly, rather than where it's just present but not really used.

Reply:
Yeah I would think something similiar to how we handle get_or_create_material type approach.